### ISSUE TITLE
Depth Extraction

### ISSUE THREAD

**Issue by GregoryBetsey** [sniklaus/3d-ken-burns#14]
Hello

I'm sorry if this is the wrong place to ask this, but I was wondering if there is a way to extract the depth map this tool creates?

Thank You

**Comment by sniklaus** [comment]
Thank you for being interested in our work! Why did using `depthestim.py` as outlined in the readme not work for you?

**Comment by GregoryBetsey** [comment]
> 
> 
> Thank you for being interested in our work! Why did using `depthestim.py` as outlined in the readme not work for you?

In short, because I don't no what I am doing. I came across you project by watching your video here https://www.youtube.com/watch?v=WrajxHHfRBA. 

I was directed here to try to use it and found that I don't know anything about python and coding. I have only been able to use the colab versions you linked in the descriptions but from what I can understand (which is very minimal) they are written to produce the auto zoom video file for an uploaded image (which I have been able to test successfully).
